README.md

```diff
@@ -1 +1,37 @@
 # Automatic Aggie Scheduler
+# Auto-Scheduler
```

Comment:
Remove this, should be replaced with the line above it

Reply:
fixed